crates/bevy_pbr/src/light.rs

```diff
@@ -15,6 +15,8 @@ pub struct Light {
     pub color: Color,
     pub fov: f32,
     pub depth: Range<f32>,
+    pub intensity: f32,
+    pub radius: f32,
```

Comment:
It would be helpful if this was called `range` instead, since I plan to use `radius` when I try to add spherical area lights.

Reply:
Done.